### PR TITLE
[Enhancement](fe): replace assertTrue(X.equals(X)) with assertEquals

### LINKED ISSUE #11916: [Enhancement] replace `assertTrue(X.equals(X))` with `assertEquals`
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Description

```java
Assert.assertTrue(column1.equals(column1));
Assert.assertFalse(column2.equals(column2));
->
Assert.assertEquals(column1, column1);
Assert.assertNotEquals(column2, column2);
```

### Solution

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by ROTBOR**
# Proposed changes

Issue Number: close #11916 

## Problem summary

Replace assertTrue(X.equals(X)) with assertEquals.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [x] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [x] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by etcZYP** [comment]
@jackwener can you plz review my pr? there are no compile errors but build failed though i use IntelliJ IDEA 
to open fe-common and there is no error prompt.
the error appears in file BitmapValueTest and line 416 210 304

File: fe/fe-core/src/test/java/org/apache/doris/load/TabletLoadInfoTest.java [hunk 1]
```diff
@@ -60,10 +60,10 @@ public void testSerialization() throws Exception {
 
         Assert.assertEquals("hdfs://host:port/dir", tabletLoadInfo1.getFilePath());
         Assert.assertEquals(1L, tabletLoadInfo1.getFileSize());
-
-        Assert.assertTrue(tabletLoadInfo1.equals(tabletLoadInfo));
-        Assert.assertTrue(rTabletLoadInfo0.equals(tabletLoadInfo0));
-        Assert.assertFalse(rTabletLoadInfo0.equals(tabletLoadInfo1));
+
```
Comment (jackwener) [inline fe/fe-core/src/test/java/org/apache/doris/load/TabletLoadInfoTest.java:63]: redundant blank.

**Comment by jackwener** [comment]
please also revert changes in `fe/fe-common/src/test/java/org/apache/doris/common/io/BitmapValueTest.java`

Because `BitmapValue` doesn't implement `equals() and hashcode()`

File: fe/fe-core/src/test/java/org/apache/doris/load/TabletLoadInfoTest.java (hunk 1 above)
Comment (jackwener) [reply fe/fe-core/src/test/java/org/apache/doris/load/TabletLoadInfoTest.java:63]: It will make build failed

Comment (jackwener) [reply fe/fe-core/src/test/java/org/apache/doris/load/TabletLoadInfoTest.java:63]: image

**Comment by etcZYP** [comment]
done but build failed still

**Comment by etcZYP** [comment]
done but still build failed

**Comment by jackwener** [comment]
Current master exit bug, wait for bugfix merge😂

**Comment by etcZYP** [comment]
apache master exit bug or mine?

**Comment by jackwener** [comment]
You need rebase the master.

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35 seconds
 load time: 633 seconds
 storage size: 17122870560 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221226145145_clickbench_pr_69015.html

**Review by jackwener — APPROVED** [review]
LGTM

**Pull request merged**